RULES/core/flags.go

```diff
@@ -225,31 +224,31 @@ func initializeFlag(flag flagInterface, name string, isInitialized *bool) {
 	Fatal("flag '%s' has disallowed value '%s'", name, info.Value)
 }
 
-func lockAndGetFlags() map[string]flagInfo {
+func lockAndGetFlags(storePersistedFlags bool) map[string]flagInfo {
```

Comment:
it's not just "store". Setting it to `true` will also _load_ the existing value.
`persistFlags`?

Reply:
I'm not sure we are on the same page here. This flag only controls whether the file is written or not. The file is always loaded, and if it is empty or does not exist, there are no persistent flags. See `getFlagValues` below